components/shared/stylo_taffy/src/convert.rs

```diff
@@ -0,0 +1,566 @@
+// Copyright 2013 The Servo Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
```

Comment:
The year is wrong here.

This is your code, so you have the ability to choose a license for it and also relicense it. When contributing this to Servo, you should use the license the project uses (MPL). If you make this a separate crate you can always relicense it.